src/postgres.rs

```diff
@@ -287,6 +267,23 @@ mod test {
         }
     }
 
+    pub static TEST_DECIMALS: &[&str; 14] = &[
+        "3950.123456",
+        "3950",
+        "0.1",
+        "0.01",
+        "0.001",
+        "0.0001",
+        "0.00001",
+        "0.000001",
+        "1",
+        "-100",
+        "-123.456",
+        "119996.25",
+        "1000000",
+        "9999999.99999",
```

Comment:
It'd be great to add those extra numbers on here too:

```
        "12340.56789",
        "79228162514264337593543950335", // 0xFFFF_FFFF_FFFF_FFFF_FFF_FFFF
        "4951760157141521099596496895", // 0x0FFF_FFFF_FFFF_FFFF_FFF_FFFF
        "4951760157141521099596496896", // 0x1000_0000_0000_0000_0000_0000
        "18446744073709551615",
        "-18446744073709551615"
```

`NUMERIC(26,6)` will be too small for some of those - one workaround is to just make it `NUMERIC` or to break the tests into two and have the second set use a larger PG scale.

Reply:
done